pyboy/api/sprite.py

```diff
@@ -116,15 +116,18 @@ def __init__(self, mb, sprite_index):
             The state of the bit in the attributes lookup.
         """
 
-        self.attr_palette_number = _bit(attr, 4)
+        if self.mb.cgb:
+            self.attr_palette_number = attr & 0x3
```

Comment:
Thank you, I had completely missed that. Generally the API has not been fully updated for CGB.

I think it should actually be `attr & 0x7`, as it's 3 bits https://gbdev.io/pandocs/OAM#byte-3--attributesflags

And I see that the bank attribute is missing too

Reply:
Yeah i was tired when i found it, i was thinking 3 bits and i wrote 3 lol.

Reply:
Take a look at the bank number var i added and see if that implementation makes sense to you.

Reply:
Perfect! The system I use for docs is a little weird, so I added a commit to fix that. The docstring has to come right after the variable, and it cannot be in an `if`-statement.

I also almost fell for the 3 vs. 7 haha